test/metabase/test.clj

```diff
@@ -42,12 +43,20 @@
             [metabase.util :as u]
             [pjstadig.humane-test-output :as humane-test-output]
             [potemkin :as p]
+            [test-plus.core :as test-plus]
             [toucan.db :as db]
             [toucan.util.test :as tt]))
 
 (humane-are/install!)
 (humane-test-output/activate!)
 
+(when config/is-dev?
```

Comment:
Is this here as opposed to in `dev` to make it available for running test-only via the clojure CLI?

Reply:
yea. but I guess we could just load it in the dev ns. let's do that

Reply:
moved it to dev https://github.com/metabase/metabase/pull/26600/commits/5c794f07727f102b58ce0c0d229dafe04891e21c